.github/workflows/linux.yml

```diff
@@ -16,44 +17,35 @@ permissions:
   contents: read
 
 jobs:
+
+  ########################
+  # linux-release-serial #
+  ########################
+
   linux-release-serial:
     # simple serial release build using g++
 
     name: linux release serial
-    runs-on: [ubuntu-20.04]
+    runs-on: [ubuntu-22.04]
 
     # only run on 'push' or if the 'pull_request' is not a draft:
     if: ${{ github.event_name == 'push' || !github.event.pull_request.draft }}
 
+    container:
+      image: dealii/dependencies:jammy
+      options: --user root
+
     steps:
     - uses: actions/checkout@v3
-    - name: setup
-      run: |
-        ./contrib/utilities/download_clang_format
     - name: info
       run: |
         g++ -v
         cmake --version
-    - name: modules
-      run: |
-        sudo apt-get install g++-10
-    - name: Build Boost
-      id: boost
-      uses: egor-tensin/build-boost@v1
-      with:
-        version: 1.74.0
```

Comment:
Is there a particular reason to use this boost version? What is in the image?

Reply:
There were C++20 issues with out bundled boost version, see https://github.com/dealii/dealii/pull/13535#discussion_r829365480. It seems like they no longer manifest in `22.04`.